cmd/subnetcmd/upgradecmd/vm.go

```diff
@@ -249,13 +250,16 @@ func updateVMByNetwork(sc models.Sidecar, targetVersion string, networkToUpgrade
 	}
 }
 
-func updateToCustomBin(_ string, _ models.VMType, _ models.Sidecar, _ string) error {
-	// get path
+func updateToCustomBin(subnetName string, sc models.Sidecar, networkToUpgrade string) error {
```

Comment:
This is a command line flag, so I don't think we can really get around this being a global. We could make this function explicitly take an extra parameter, but I don't think that addresses the root of your concern. I'm not too worried about this because we use this pattern throughout the codebase whenever we take command line arguments.

Reply:
With this addressed LGTM

Reply:
addressed